ext/drstatecmp/drstatecmp.h

```diff
@@ -0,0 +1,99 @@
+/* **********************************************************
+ * Copyright (c) 2021 Google, Inc.   All rights reserved.
+ * **********************************************************/
+
+/*
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * * Neither the name of Google, Inc. nor the names of its contributors may be
+ *   used to endorse or promote products derived from this software without
+ *   specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL GOOGLE, INC. OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+ * DAMAGE.
+ */
+
+#ifndef _DRSTATECMP_H_
+#define _DRSTATECMP_H_
+
+/**
+ * @file drstatecmp.h
+ * @brief Header for DynamoRIO Machine State Comparison Library
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * \addtogroup drstatecmp Machine State Comparison Library
+ */
+/*@{*/ /* begin doxygen group */
+
+/** Success code for each drstatecmp operation. */
+typedef enum {
+    DRSTATECMP_SUCCESS,                   /* Operation succeeded. */
+    DRSTATECMP_ERROR,                     /* Operation failed. */
+    DRSTATECMP_ERROR_ALREADY_INITIALISED, /* DRSTATECMP can only be initialised once. */
+    DRSTATECMP_ERROR_NOT_INITIALIZED,     /* Operation failed: not initialized. */
```

Comment:
Be consistent: you have INITIALISED on one line (British spelling) and INITIALIZED on the next, in the code and the comments.